### PR TITLE
Feature/stats count

### PR DESCRIPTION
**Pull request by RuvimboRue**
Adding the stats and quote section

### CODE REVIEW

File: Frontend/components/statsCount.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        </div>
+      </div>
+      <div className="text-center mb-8 sm:mb-2">
+        <div style={{ fontSize: "25px" }} className="font-bold">
+          <CountUp start={0} end={300} duration={3} />
+          +
+        </div>
+        <div className="font-bold mt-2">
+          <p>Community events organised</p>
+        </div>
```
Comment (SirSanctified) [inline Frontend/components/statsCount.jsx:23]: Screen Shot 2023-11-13 at 11 36 32

Comment (SirSanctified) [reply Frontend/components/statsCount.jsx:23]: I have resolved this one

Comment (RuvimboRue) [reply Frontend/components/statsCount.jsx:23]: Okay thank you

Comment (SirSanctified) [reply Frontend/components/statsCount.jsx:23]: Will be pushing the changes any minute

**Review by SirSanctified — APPROVED** [review]

**Pull request merged**